spyder/app/mainwindow.py

```diff
@@ -1556,115 +1579,142 @@ def setup_default_layouts(self, index, settings):
         global_hidden_toolbars = [self.source_toolbar, self.edit_toolbar,
                                   self.search_toolbar]
         # Layout definition
-        # layouts are organized by columns, each colum is organized by rows
-        # widths have to add 1.0, height per column have to add 1.0
+        # --------------------------------------------------------------------
+        # Layouts are organized by columns, each column is organized by rows

```

Comment:
Is this line missing a period? And the next one should start with `Width`?

If not, I don't understand the meaning of this comment.

Reply:
Yes it was.